src/main/java/gregtech/api/metatileentity/MetaTileEntity.java

```diff
@@ -497,6 +497,11 @@ public boolean onCrowbarClick(EntityPlayer playerIn, EnumHand hand, EnumFacing f
      * @return true if something happened, so the tool will get damaged and animation will be played
      */
     public boolean onSoftMalletClick(EntityPlayer playerIn, EnumHand hand, EnumFacing facing, CuboidRayTraceResult hitResult) {
+        CoverBehavior behavior = getCoverAtSide(hitResult.sideHit);
+        if (behavior != null && behavior.onSoftMalletClick(playerIn, hand, hitResult) == EnumActionResult.SUCCESS) {
+            return true;
+        }
```

Comment:
Originally, this logic was for checking the entire side of a meta tile entity instead of tracing with the machine grid. After discussion, it has been removed in https://github.com/GregTechCEu/GregTech/commit/b78be251c1d22fd89fc37d9436a154bb724d7c36.